geotrek/trekking/management/commands/sync_rando.py

```diff
@@ -810,6 +793,8 @@ def handle(self, *args, **options):
             'ignore_errors': True,
             'tiles_dir': os.path.join(settings.DEPLOY_ROOT, 'var', 'tiles'),
         }
+        self.tmp_root = os.path.join(os.path.dirname(self.dst_root), 'tmp_sync_rando')
+        os.mkdir(self.tmp_root)
```

Comment:
if not os.path.exists: os.mkdir.. gerer les cas ou le fichier existe deja (exception)